public/injectEnv.js

```diff
@@ -2,7 +2,7 @@ window.injectedEnv ={
 	REACT_APP_BACKEND_GETUSERINFO_API:'https://k9dc.essential-dev.com/fence/login/',
 	REACT_APP_LOGIN_URL:'https://nci-crdc-staging.datacommons.io/user/oauth2/authorize?client_id=82pslYFJqA7auRvKYfTOK67jzQAMb8f6C33tlmZz&response_type=code&redirect_uri=https%3A%2F%2Fk9dc.essential-dev.com%2F&scope=openid%20user',
 	REACT_APP_USER_LOGOUT_URL:'https://k9dc.essential-dev.com/fence/logout',
-	REACT_APP_BACKEND_API:'http://localhost:8080/v1/graphql/',
+	REACT_APP_BACKEND_API:'https://api.bento-tools.org/v1/graphql/',
```

Comment:
Intentional change?

Reply:
what do you think? as we are no longer running the backend in our local thought to keep prod api as env variable.

Reply:
Cool. Makes sense